packages/lib/serverConfig.ts

```diff
@@ -16,6 +18,9 @@ function detectTransport(): SendmailTransport.Options | SMTPConnection.Options |
         pass: process.env.EMAIL_SERVER_PASSWORD,
       },
       secure: port === 465,
+      tls: {
+        rejectUnauthorized: isENVDev ? false : true,
+      },
```

Comment:
This was necessary for me to be able to run maildev and recieve mails on localhost, it only disables rejecting unathorized in dev, not prod.